src/main/java/dto/Transaction.java

```diff
@@ -0,0 +1,19 @@
+package dto;
+
+import lombok.Getter;
+import model.OperationEnum;
+
+@Getter
+public class Transaction {
+
+    private final OperationEnum operation;
+    private final String product;
+    private final int quantity;
+
+    public Transaction(OperationEnum operation, String product, int quantity) {
+        this.operation = operation;
+        this.product = product;
+        this.quantity = quantity;
+    }
+
```

Comment:
Optional: Please remove empty line before the end of class and add the empty line after the end of class (this is not mandatory, just a common style)

Reply:
Thank you, done.